src/http.rs

```diff
@@ -196,7 +196,7 @@ pub enum HttpWriter<W: Writer> {
     /// A Writer for when Content-Length is set.
     ///
     /// Enforces that the body is not longer than the Content-Length header.
-    SizedWriter(W, usize),
+    SizedWriter(W, u64),
```

Comment:
Likewise, if your server has files bigger than 4gb (which is becoming more common), then a 32bit server won't be able to serve them.